modules/ROOT/pages/sso-use-case-1.adoc

```diff
@@ -3,26 +3,23 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Use case one includes all of the instructions for setting up an identity provider for your portal when no other identity provider is not configured in Anypoint. You want to configure an identity provider as a login option for your portal. Since there is no identity provider configured in Anypoint, you don’t need to map the identity provider for the portal to the identity provider in Anypoint since the default identity provider can be used for the portal. This enables every Salesforce user to have an identity in Anypoint.
+Use case one includes all of the instructions for setting up an identity provider for your portal when no other identity provider is not configured in Anypoint Platform. You want to configure an identity provider as a login option for your portal. Since there is no identity provider configured in Anypoint Platform, the default identity provider is used to map the identity provider for the portal to the identity provider in Anypoint Platform. This enables every Salesforce user to have an identity in Anypoint.
```

Comment:
First paragraph in this topic: Initial-cap "is" and remove "Already."

Reply:
done